scss/_patterns_chip.scss

```diff
@@ -71,8 +71,38 @@
   @include vf-chips-information;
 }
 
+@mixin vf-chip-interactive(
+  $color-background-hover: $colors--theme--background-neutral-hover,
+  $color-border-hover: $colors--theme--border-neutral,
+  $color-background-active: $colors--theme--background-neutral-active,
+  $color-border-active: $colors--theme--border-neutral
+) {
+  &:hover {
+    background-color: $color-background-hover;
+    border-color: $color-border-hover;
+    text-decoration: none;
+  }
+
+  &[aria-pressed='true'],
+  &.is-selected,
+  &:active {
+    background-color: $color-background-active;
+    border-color: $color-border-active;
+  }
+}
+
 // Default notification styling
 @mixin vf-chips-default {
+  // stylelint-disable selector-max-type -- allow button selector for interactive chips
+
+  // Status-color chips all use the button pattern mixin to set their colors, so they already have most button styling.
+  // We need to add the default button styling to the link default chip as well.
+  a.p-chip {
```

Comment:
As this is more specific than `.p-chip` would it make sense to move it below? Somewhere alongside the `button.p-chip` (this seems very similar).

Reply:
Done